06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexes = {};
+  }
+
+  /**
+   * Validate File
+   * @function
+   * @param {string} filedata
+   * @return {Boolean} true or false
+   */
+  static validate(filedata) {
```

Comment:
I believe the pattern is that all documents in each file being uploaded should contain `title` and `text` properties.